### PR TITLE
Added failing test for TDD

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/45271
fixes: https://github.com/demisto/etc/issues/45272

## Description
Fixed validations not adjusted to work with CONTRIBUTORS file.

## Must have
- [x] Tests

### CODE REVIEW

**Comment by ShahafBenYakir** [comment]
Failed UT that simulate a failing validation: 
https://app.circleci.com/pipelines/github/demisto/demisto-sdk/13291/workflows/e39004e5-5f48-4fca-b43f-57538a14aaa0/jobs/72449

File: demisto_sdk/commands/validate/tests/validators_test.py [hunk 1]
```diff
@@ -801,17 +802,17 @@ def test_validate_no_missing_release_notes__missing_rn_dependent_on_api_module(s
...
-                               {
-                                   'name': integration2.name,
-                                   'file_path': integration2.path,
-                                   'pack': pack2_name,
-                                   'api_modules': api_script1.name
-                               }
-                               }
-                          ]
-                          }
+            [
```
Comment (yaakovi) [inline demisto_sdk/commands/validate/tests/validators_test.py:805]: will fix lint errors
```
        id_set_content = {
            'integration': [
```

File: demisto_sdk/commands/validate/validate_manager.py [hunk 2]
```diff
@@ -574,6 +574,10 @@ def run_validations_on_file(self, file_path, pack_error_ignore_list, is_modified
         elif file_type == FileType.JOB:
             return self.validate_job(structure_validator, pack_error_ignore_list)
 
+        elif file_type == FileType.CONTRIBUTORS:
+            # This is temporarily - need to add a proper contributors validations
```
Comment (yaakovi) [inline demisto_sdk/commands/validate/validate_manager.py:578]: open an issue so we can track and prioritize it

**Review by yaakovi — APPROVED** [review]

File: demisto_sdk/commands/validate/tests/validators_test.py (hunk 1 above)
Comment (ShahafBenYakir) [reply demisto_sdk/commands/validate/tests/validators_test.py:805]: :yaakovi:

**Pull request merged**